### ISSUE TITLE
Other __2vecs

### ISSUE THREAD

**Issue by milcent** [vector-ai/vectorhub#3]
Congratulations and thank you for this amazing initiative!
How about adding `cat2vec`, for categorical, tabular data, and `node2vec`, for network graphs?
Best wishes,
Milcent

**Comment by JackyKoh** [comment]
Great issue, these are definitely 2 model types that we will be adding very soon.

**Comment by SilenceBlue** [comment]
Congratulations and thanks for your contribution, which is very useful in industry！
Do you have a video2vec Model in the future? For example, TSN+attentionLSTM is used to extract video features

**Comment by JackyKoh** [comment]
@SilenceBlue Yeah that's actually one of the priority 2vec model types we are adding.

**Comment by SilenceBlue** [comment]
It's awesome! I'm looking forward to your video2vec models extremely！
